proposals/0002-root-signature-in-clang.md

```diff
@@ -393,6 +393,24 @@ stored as metadata nodes, identified by named metadata. The metadata format
 itself is a straightforward transcription of the in-memory data structure - so
 it is a list of root elements.
 
+While the attribute is attached to a function, the metadata collects all the
+root signatures together, with the initial metadata associating the root
+signatures with functions.
+
+Example for same root signature as above:
+
+```llvm
+!dx.rootsignatures = !{!2} ; list of function/root signature pairs
+!2 = !{ptr @main, !3 } ; function, root signature
```

Comment:
```suggestion
!2 = !{ ptr @main, !3 } ; function, root signature
```
Not sure if needed, but everywhere else seems to have a space. There is another instance if this is the case on line 524